### DISCUSSION TITLE
How to check HF part of a Jumper T-Lite CC2500 module?

### DISCUSSION THREAD

**Discussion by onki69** [pascallanger/DIY-Multiprotocol-TX-Module#637] [General]
Hi,

I am having a problem with a new cc2500 type Jumper T-Lite.
I can't bind any receiver (they work with other TX) and I see nothing in the spectrum feature.
Firmware can be flashed from the Jumper successfully so the µC seems to be working.

Is there a way to measure (with a scope) anything at the CC2500 to check if the HF-part is working?

Currentwise I see no difference if I start binding, range test or low power mode. Only module on/off shows approx. 40-50mA difference (I guess this is just the µC.

Any ideas?

Onki

**Comment by onki69** [comment]
I was able to get the T-Lite CC2500 up and running.
The problem is the firmware wizard on the multi-module.org website.

When selecting the Jumper T-Lite CC2500 the regular firmware will be shown.
But this was not working for me. After choosing the 64kb type firmware the module works OK to me.

Should I open a bug to have this fixed or is my T-Lite a special thing that needs the 64k variant.?

Regards Onki

**Comment by benlye** [comment]
I think this is the first T-Lite I've heard of which has a 64KB MCU. I can certainly update the download site to offer the 64KB bin file for it, same as we do for the TX12.

**Comment by InsaneMonkey** [comment]
Yes, DO IT. I have spent about 2 weeks to get my Jumper T-Lite to work and then i found this. And it works Again. So pleas do it for other people with the same problem. : )
